builtin/providers/google/resource_compute_instance_test.go

```diff
@@ -417,6 +418,31 @@ func TestAccComputeInstance_subnet_custom(t *testing.T) {
 	})
 }
 
+func TestAccComputeInstance_subnet_xpn(t *testing.T) {
+	var instance compute.Instance
+	var instanceName = fmt.Sprintf("instance-test-%s", acctest.RandString(10))
+	var xpn_host = os.Getenv("GOOGLE_XPN_HOST_PROJECT")
```

Comment:
@stack72 This new feature (cross-project networking) introduces the need for a second GCP project for acceptance tests. How should we work to get this introduced to the acc suite that Travis runs?